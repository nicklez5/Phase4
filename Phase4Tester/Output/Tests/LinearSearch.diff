--- SelfTestCases/LinearSearch.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/LinearSearch.out	2020-03-14 21:50:36.722514280 -0700
@@ -9,8 +9,8 @@
 18
 9999
 0
-1
-1
+0
+0
 0
 55
 
