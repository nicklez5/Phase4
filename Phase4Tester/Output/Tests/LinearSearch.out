Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/LinearSearch.compiler.out line 207 column 11: "2": operand is of incorrect type

Processing terminated due to errors.
